js/fetchDashBoard.js

```diff
@@ -87,6 +87,32 @@ async function addContactFunction() {
     } catch (error) {
       document.getElementById("add-result").textContent = `Error: ${error.message}`;
     }
+
+    //messing field message
+    let str="Required:\n ";
```

Comment:
is this to check if field is missing when try to add new contact?

You should check this before the fetch operation, and if any field is missing, immediately return the function without having the fetch the data from backend